dev-dotnet/dotnet-reference-packages/dotnet-reference-packages-1.0.0_p20210210.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit check-reqs
+
+# Disclaimer: While a bunch of generated .il (.net assembly) files can be
+#  considered not to be *source code* per se, none of the files here contain
+#  any actual code. Only symbols/references for a variety of libs.
+
+DESCRIPTION=".NET Source-build Reference Packages"
+HOMEPAGE="https://github.com/dotnet/source-build-reference-packages/"
+
+BOOT_PV="5.0.202"
+BOOT_ARTIFACTS_PV="0.1.0-${BOOT_PV}-1088763-20210414.1"  # See eng/Versions.props
+BOOT_SRC_URI="https://dotnetcli.azureedge.net/dotnet/Sdk/${BOOT_PV}/dotnet-sdk-${BOOT_PV}-linux-x64.tar.gz
+	https://dotnetcli.azureedge.net/source-built-artifacts/assets/Private.SourceBuilt.Artifacts.${BOOT_ARTIFACTS_PV}.tar.gz
+"
+
+MY_PN=source-build-reference-packages
+MY_PV=bcdc58a2ae9c2c06d769acfb8375d27adf5b2332
+SRC_URI="https://github.com/dotnet/source-build-reference-packages/archive/${MY_PV}.tar.gz -> ${P}.tar.gz
+	!system-bootstrap? ( ${BOOT_SRC_URI} )
+"
+S="${WORKDIR}/${MY_PN}-${MY_PV}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="system-bootstrap"
+BDEPEND="
+	system-bootstrap? ( >=dev-dotnet/dotnet-sdk-${BOOT_PV} )
+"
+
+CHECKREQS_DISK_BUILD="14G"
+CHECKREQS_DISK_USR="400M"
+
+dotnet_unpack() {
```

Comment:
Ideally, we'd make this a 'local' (not really) function within src_unpack, although you mentioned that may make it a pain for copying b/t ebuilds. We can consider an eclass (or hijacking one of the existing ones with a revision, maybe) later.

Reply:
For `dotnet_find`, yeah, I agree there should be an eclass for it, but I'm not convinced dotnet_unpack won't change with future versions of these two bootstrap ebuilds.